### PR TITLE
Add user comment to contact form 

### PR DESCRIPTION
**Pull request by baha-a**
Fixes:
- https://github.com/buerokratt/Buerokratt-Chatbot/issues/1#issuecomment-1938799114

### CODE REVIEW

File: src/translations/et/common.json [hunk 1]
```diff
@@ -71,13 +74,5 @@
   "widget.action.confirm": "Kinnita",
   "widget.action.download-chat": "Lae vestlus alla",
   "widget.action.forward-chat": "Edasta vestlus e-mailile",
-  "widget.action.forward-email": "Teie e-postiaadress",
```
Comment (1AhmedYasser) [inline src/translations/et/common.json:74]: Please revert waiting time removed translations as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/translations/en/common.json [hunk 2]
```diff
@@ -71,13 +74,5 @@
   "widget.action.confirm": "Confirm",
   "widget.action.download-chat": "Download conversation",
   "widget.action.forward-chat": "Forward to an email",
-  "widget.action.forward-email": "Your email address",
-  "widget.form.phone": "Your phone number",
-  "widget.form.email": "Your email address",
```
Comment (1AhmedYasser) [inline src/translations/en/common.json:76]: Please revert waiting time removed translations as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/info-icon.tsx
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/info-icon.tsx:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/notification-message.tsx
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/notification-message.tsx:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/waiting-time-notification-form-success.tsx
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/waiting-time-notification-form-success.tsx:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/waiting-time-notification-form.tsx
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/waiting-time-notification-form.tsx:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/waiting-time-notification.module.scss
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/waiting-time-notification.module.scss:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/waiting-time-notification.test.tsx
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/waiting-time-notification.test.tsx:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

File: src/components/waiting-time-notification/waiting-time-notification.tsx
Comment (1AhmedYasser) [inline src/components/waiting-time-notification/waiting-time-notification.tsx:1]: Please revert waiting time notification as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445

**Review by 1AhmedYasser — CHANGES_REQUESTED** [review]

**Comment by 1AhmedYasser** [comment]
in here 

```
 const submitForm = async (e: React.MouseEvent<HTMLButtonElement>) => {
    e.preventDefault();
    if (validateInput(endUserContacts)) {
      await WidgetService.sendContactInfo(
        chatId ?? "",
        endUserContacts.mailAddress,
        endUserContacts.phoneNr
      ).catch(console.error);
      const newMsg = getContactFormFulfilledNewMessage(
        endUserContacts,
        chatId,
        contactMsgId,
        t
      );
      dispatch(sendMessageWithNewEvent(newMsg));
      dispatch(setShowUnavailableContactForm(false));
      newMsg.content = "";
      dispatch(sendMessagePreview(newMsg));
      dispatch(
        endChat({
          event: CHAT_EVENTS.UNAVAILABLE_CONTACT_INFORMATION_FULFILLED,
          isUpperCase: false,
        })
      );
      dispatch(setShowUnavailableContactForm(false));
    }
  };
```

you should check if the user left a comment, if he does then send a message before calling `dispatch(endchat)`
the message content should be something as follows

`content: 'User Question:\n${question}'`

**Comment by baha-a** [comment]
Requested changes resolved.

**Review by 1AhmedYasser — APPROVED** [review]

**Pull request merged**